### ISSUE TITLE
Pull request template

### ISSUE THREAD

**Issue by derek-watson** [twg/devour#21]
I'd like to consider removing the pull request template from this project. I find clear and concise english to be much more expressive way to describe the intention of your PR, and in many cases the template is getting in the way of clarity.

Here is a recent full-screen desktop screenshot of a recent pull request. In my mind that amount of screen real estate should contain a lot more information.

![screen shot 2016-05-07 at 1 11 21 pm](https://cloud.githubusercontent.com/assets/334285/15093425/7f9ce778-1455-11e6-965e-7d81f8f442a3.png)

**Comment by Emerson** [comment]
I'm fine with it – it's pretty big at the moment 👍

**Comment by billxinli** [comment]
Perhaps we should rearrange the important bits on top, and make the headers smaller?

**Comment by Emerson** [comment]
👍 for making it less verbose

**Comment by randym** [comment]
+1

From an outside perspective, Documents and Bug/Ticket tracker information may be included as part of the 'Why' in 'What Changed & Why'

Priority is too subjective without some kind of definition and probably best left to the modules maintainers.

**Comment by raizyr** [comment]
I agree with randym.   The pull request template seems very internally focused.  Much of it make no sense for an open source project.

For example:

> Note anything that is waiting on other departments or team members.

Um, what departments or team members?  I just found this repo via http://jsonapi.org/implementations/.

>Mention people who would be interested in the changeset:
>...
>Designer(s)
>Product manager (if they’re interested)

Product manager? Designer? I have no idea who they are nor why they'd care about my issues with the code in this repo.

Pull Request templates can be useful, however, here is one that I find helpful: https://github.com/mxstbr/react-boilerplate/blob/master/.github/PULL_REQUEST_TEMPLATE.md
